### DISCUSSION TITLE
Umbraco 14 & 15

### DISCUSSION THREAD

**Discussion by patrickdemooij9** [patrickdemooij9/SeoToolkit.Umbraco#292] [General]
This is a discussion to track the development of migrating to Umbraco 14. This should hopefully help anyone that wants to know how far along the migration is.

All the development is done on dev/main and this will most likely become the main branch when the migration has been finished.

Modules to migrate:
- [x] RobotsTxt
- [x] ScriptManager
- [x] Redirects
- [x] MetaFields
- [x] Sitemap
- [x] SiteAudit

Some features still missing:
- [x] Redirect import
- [x] Redirect to media (as I haven't gotten the property editor working yet)
- [x] Properly handle multiple cultures

BETA Version: https://github.com/patrickdemooij9/SeoToolkit.Umbraco/releases/tag/4.0.0-beta1

**Comment by The-algar** [comment]
Really looking forward to the V14 Sitemap

**Comment by koenvanras** [comment]
Any update on the progress (since I saw a few commits on dav/main suggesting some features are done)? Would love to start using the package for some of my upgraded projects :)

**Reply by patrickdemooij9** [reply]
Hi @koenvanras!
The latest update is that all but 2 modules are finished. Last ones being the SiteAudit and MetaFields. I am hoping that I can have a first version available before the new year!

**Comment by koenvanras** [comment]
Hi @patrickdemooij9,
Do you happen to have any update on this?

**Reply by patrickdemooij9** [reply]
Hi @koenvanras 
Yes, unfortunately (as you probably noticed) I wasn't able to get the first version out before the new year. Currently I am left with the MetaFields content view and some smaller todos. I hope to have a first beta version this month. I'll then let it be for a bit while people test and then I can release the first official version for both v14 & v15.

**Reply by koenvanras** [reply]
Hi @patrickdemooij9,
Sounds great! I'm looking forward to seeing the beta version and will definitely help with testing if that is possible!

**Comment by patrickdemooij9** [comment]
Hi all!
I have just released the first Beta version for Umbraco 15. Check out the release here for more information: https://github.com/patrickdemooij9/SeoToolkit.Umbraco/releases/tag/4.0.0-beta1
